demo/python/multiclient-cluster-remote-control/client1/client1.py

```diff
@@ -75,5 +72,6 @@ def run(channel_addr, sym_key_file, priv_key_file, cert_file):
 
     args = parser.parse_args()
 
-    channel_addr = str(args.ip_addr) + ":50051" 
+    # Connect to the orchestrator
+    channel_addr = str(args.ip_addr) + ":50052" 
```

Comment:
Let's try to make this configurable? In particular, you can replace lines 72-76 with this block of code:

```
parser.add_argument("--port", help="orchestrator port", default=50051)
args = parser.parse_args()
channel_addr = str(args.ip_addr) + ":" + str(args.port)
```
We'll also need to do the same thing `client2.py`.

And you'll have to add the `--port 50052` option to `run.sh` in both `client1/` and `client2/`.

Reply:
Done!